docs/trusted-rpc-interface.md

```diff
@@ -0,0 +1,63 @@
+# Trusted Rpc Interface
+This document contains information about the trusted worker json-rpc interface. The trusted worker server is a tls websocket directly running in the enclave.
+
+The server expects an json-rpc call of the following format:
+
+`{"jsonrpc": "2.0", "method": "author_pendingExtrinsics", "params": ["5Ki5bf4dcY9eyrqBRe6Xbr5accvo42XZb86eXv5mkTJo"], "id": 1}`
+
+The workers methods are aligned to match as much as sensible the substrate rpc calls: https://docs.substrate.io/v3/runtime/custom-rpcs/
```

Comment:
I don't understand the sentence. What does this mean?

Reply:
Better now?

Reply:
very much!